docs/modules/ROOT/pages/index.adoc

```diff
@@ -1139,6 +1139,14 @@ public class Application extends HxController {
 <4> Only Hx requests are allowed, else it will fail with a BAD_REQUEST error
 <5> Flag the response with an https://htmx.org/reference/#response_headers[htmx response header]
 
+For CSRF Security, you need a form parameter with the CSRF Token. By adding this 👇 when doing a hx-post/put/delete, The Hx requests will be sent with the CSRF parameter:
+[source,html]
+----
+<div hx-post"/hello" hx-vals='{"{inject:csrf.parameterName}": "{inject:csrf.token}"}'>
+---
+
+NOTE: There is a ongoing issue to allow using a header instead of a form parameter (https://github.com/quarkusio/quarkus/issues/34513), this way it will be possible to have a `hx-headers` on the <body> to make all hx requests secured with CSRF.
```

Comment:
Andy, not sure this note has to be part of Renarde docs

Reply:
Why?

Reply:
@ia3andy Sorry, missed it, well, there is a commitment in this doc message that the linked issue will be addressed - in fact it will be, but in general, it is not guaranteed